getstation/website-v3
# [APP-699] Refactor `delete-account` page to use our authentication UI rather than Auth0's one

Refactor `delete-account` to use our Authentication UI rather than Auth0's one.

## Code change summary
Rely on https://github.com/getstation/authentication-ui/pull/10
- revert https://github.com/getstation/website-v3/commit/9112ab6940948b5e0557ae18947d7000a3b22cc0
- replace Auth0Lock by `@getstation/authentication`'s `<ProvidedAuthenticationBx/>`
- => add env variables
- => remove now unnecessary background images + add a bit of styling
- => fetch current user info via the API

## Test instruction
- create a fake account (with tempmail)
- go to https://deploy-preview-40--station-website.netlify.com/delete-account/
- follow flow to delete the fake account
- in[ Forest admin (prod)](https://app.forestadmin.com/Station/Production/Operations/data/1819021/index), you should see your deletion request

## Todos
- [ ] Merge  https://github.com/getstation/authentication-ui/pull/10 and install version 1.0.0 here
